packages/presentation/lib/screen/home/widgets/home_grid_view.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:presentation/config/theme/app_colors.dart';
+import 'package:presentation/screen/home/widgets/movie_content.dart';
+import 'package:presentation/screen/home/widgets/movie_rating.dart';
+import 'package:presentation/screen/home/widgets/movie_title.dart';
+
+import 'package:presentation/utils/image_path.dart';
+
+class HomeGridView extends StatelessWidget {
+  const HomeGridView({
+    Key? key,
+    required this.nowShowingData,
+    required this.blocFunctions,
+  }) : super(key: key);
+
+  final nowShowingData;
+  final blocFunctions;
```

Comment:
to the comment about the correct use of your "HomeData", you would not have to create such bad variables